package.json

```diff
@@ -17,100 +18,102 @@
+    "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-macros": "^3.1.0",
-    "babel-plugin-polyfill-corejs3": "^0.6.0",
+    "babel-plugin-polyfill-corejs3": "^0.8.1",
     "babel-plugin-react-transform": "^3.0.0",
-    "babel-preset-env": "^1.6.0",
-    "chromatic": "^6.15.0",
+    "babel-preset-env": "^1.7.0",
+    "chromatic": "^6.17.4",
     "copy-webpack-plugin": "^11.0.0",
-    "css-loader": "^6.7.1",
-    "css-minimizer-webpack-plugin": "^4.0.0",
+    "css-loader": "^6.7.3",
+    "css-minimizer-webpack-plugin": "^5.0.0",
     "eslint": "^8.40.0",
     "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-mdx": "^2.0.5",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-mdx": "^2.1.0",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-spellcheck": "0.0.20",
     "eslint-plugin-storybook": "^0.6.12",
     "extract-loader": "^5.1.0",
     "foundation-sites": "^6.7.5",
     "glob": "^8.1.0",
-    "html-entities": "^2.3.2",
-    "lodash-webpack-plugin": "^0.11.5",
-    "mini-css-extract-plugin": "^2.7.0",
-    "patch-package": "^6.4.7",
+    "html-entities": "^2.3.3",
+    "lodash-webpack-plugin": "^0.11.6",
+    "mini-css-extract-plugin": "^2.7.5",
+    "patch-package": "^7.0.0",
     "path": "^0.12.7",
     "postcss-cli": "^10.1.0",
-    "postcss-loader": "^7.0.2",
-    "prettier": "^2.8.0",
+    "postcss-loader": "^7.3.0",
+    "prettier": "^2.8.8",
     "prettier-eslint": "^15.0.1",
     "prettier-stylelint": "^0.4.2",
     "react-syntax-highlighter": "^15.5.0",
     "remove-files-webpack-plugin": "^1.5.0",
     "resolve-url-loader": "^5.0.0",
-    "sass": "^1.56.1",
-    "sass-loader": "^13.2.0",
-    "storybook-addon-rtl": "^0.4.4",
-    "style-loader": "^3.3.1",
-    "stylelint": "^14.15",
+    "sass": "^1.62.1",
+    "sass-loader": "^13.2.2",
+    "storybook": "^7.0.18",
+    "storybook-addon-rtl": "^0.5.0",
+    "style-loader": "^3.3.2",
+    "stylelint": "^15.6",
     "stylelint-config-prettier": "^9.0.5",
-    "stylelint-config-recommended-scss": "^8.0",
-    "stylelint-config-sass-guidelines": "^9.0.1",
-    "stylelint-config-standard": "^29.0.0",
-    "stylelint-scss": "^4.4",
+    "stylelint-config-recommended-scss": "^11.0",
+    "stylelint-config-sass-guidelines": "^10.0.0",
+    "stylelint-config-standard": "^33.0.0",
+    "stylelint-scss": "^5.0",
     "svg-spritemap-webpack-plugin": "^4.5.0",
     "svgo": "^3.0.2",
-    "svgo-loader": "^3.0.1",
-    "webpack": "^5.76.1",
-    "webpack-cli": "^5.0.1",
+    "svgo-loader": "^4.0.0",
+    "webpack": "^5.82.1",
+    "webpack-cli": "^5.1.1",
     "webpack-fix-style-only-entries": "^0.6.1"
   },
   "dependencies": {
+    "@storybook/mdx1-csf": "^1.0.0",
+    "core-js": "^3.30.2",
```

Comment:
Correct, they don't need to be in dependencies. I've moved them into the dev dependencies. 

core-js 3 was required by the babel updates. core-js 2 just wasn't working and wasn't updating with the necessary dependency updates, so I had to specifically add v3 into the list to finally get it to start compiling.

@storybook/mdx1-csf is required to make some storybook6 mdx1 formats compile into storybook7 csf format. Otherwise a 100% complete rewrite of all components would be necessary.